src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -488,6 +500,14 @@ private String parseOutput(String llmOutput, String indexName) {
         }
         ppl = ppl.replace("`", "");
         ppl = ppl.replaceAll("\\bSPAN\\(", "span(");
+        if (this.head > 0) {
+            String[] lists = llmOutput.split("\\|");
```

Comment:
Can we simply use contains method to check if llmOutput has head, if true do not modify it. This is more efficient than splitting and then check.

Reply:
but head can be the field name. Split can be the safe way.